src/backend/utils/__mocks__/indexer.js

```diff
@@ -21,7 +28,7 @@ const deletePost = (postId) => {
 };
 
 const search = (keyword = '') => {
```

Comment:
This doesn't match your new signature either: `const search = async (textToSearch, filter = 'post') => {`